yt_dlp/extractor/atvat.py

```diff
@@ -94,6 +96,11 @@ def _real_extract(self, url):
             })
 
         video_id, videos_data = list(videos['data'].items())[0]
+        if 'error' in videos_data:
+            msg = videos_data['error'].get('title')
+            if msg == 'Geo check failed':
+                raise GeoRestrictedError(msg)
+            raise ExtractorError(msg)
```

Comment:
```suggestion
        msg = try_get(videos_data, lambda x: x['error']['title'])
        if msg == 'Geo check failed':
            raise GeoRestrictedError(msg)
        elif msg:
            raise ExtractorError(msg)
```
also, should the ExtractorError have expected=True?

Reply:
I committed the suggestion. Since we don't know what other errors would be we can't really claim that they are expected imho.